### PR TITLE
Enhancement/4 vs common document files 3.0.0

### LINKED ISSUE #4: Use Virtual-Schema-Common-Document-Files-3.0.0
Add by upgrading to new vs-common-document-files version

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #4

### CODE REVIEW

File: doc/changes/changes_0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# s3-document-files-virtual-schema 0.2.0, released 2020-XX-XX
+ 
+Code name: 
+
+## Summary
+
+This Release implements the changes for the virtual-schema-common-document-files 3.0.0 release.
```
Comment (morazow) [inline doc/changes/changes_0.2.0.md:7]:
```suggestion
This release implements the changes for the virtual-schema-common-document-files 3.0.0 release.
```

File: doc/changes/changes_0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+The changes also cause an API change for the UDF definition. See the [user guide](../user_guide/user_guide.md).
+
+## Features / Enhancements
+
+* #4: Use Virtual-Schema-Common-Document-Files-3.0.0
+
+## Dependency Updates:
+
+* Updated `com.exasol:exasol-testcontainers` from 3.2.0 to 3.3.0
```
Comment (morazow) [inline doc/changes/changes_0.2.0.md:20]: You also updated `org.testcontainers` version.

**Review by morazow — APPROVED** [review]
Suggested updates to release letter.

File: doc/changes/changes_0.2.0.md (hunk 2 above)
Comment (jakobbraun) [reply doc/changes/changes_0.2.0.md:20]:
```suggestion
* Updated `com.exasol:exasol-testcontainers` from 3.2.0 to 3.3.0
* Updated `org.testcontainers:junit-jupiter` from 1.14.3 to 1.15.0
* Updated `org.testcontainers:localstack` from 1.14.3 to 1.15.0
```

**Pull request merged**